File: uf-web/src/Entity/Enumerations.php
<?php
namespace App\Entity;

enum UploadStatus: string {
    case Pending = 'PENDING';
    case Fetching = 'FETCHING';
    case Completed = 'COMPLETED';
    case Errored = 'ERRORED';
    case Cancelled = 'CANCELLED';
}

enum UploadError: string {
    case PageUnavailable = "PAGE_UNAVAILABLE";
    case StoryInaccessible = "STORY_INACCESSIBLE";
    case MarkupUnexpected = "MARKUP_UNEXPECTED";
    case Freeform = "ERROR_FREEFORM";
}

enum LogLevel: string {
    case Debug = 'DEBUG';
    case Info = 'INFO';
    case Warn = 'WARN';
    case Error = 'ERROR';
}

File: uf-uploader/source/common/src/main/kotlin/fi/sobolev/uffn/common/data/Enumerations.kt
package fi.sobolev.uffn.common.data


enum class Archive {
    AO3, FFN;
}

enum class Rating {
    K, T, M, E;
}

enum class UploadStatus {
    PENDING, FETCHING, COMPLETED, ERRORED, CANCELLED;
}

enum class UploadError {
    PAGE_UNAVAILABLE, STORY_INACCESSIBLE, MARKUP_UNEXPECTED, ERROR_FREEFORM;
}

enum class LogLevel {
    DEBUG, INFO, WARN, ERROR;
}
